### PR TITLE
MobArena Support

### PR DESCRIPTION
**Pull request by Fortifier42**
Not really tested. Unable to test with current setup.

### CODE REVIEW

File: src/main/java/net/gnomeffinway/depenizen/events/MobArena/MobArenaStartsScriptEvent.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import org.bukkit.Bukkit;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+// <--[event]
+// @Events
+// mobarena arena starts
+// mobarena <arena> starts
+//
+// @Regex ^on mobarena [^\s]+ starts
```
Comment (mcmonkey4eva) [inline src/main/java/net/gnomeffinway/depenizen/events/MobArena/MobArenaStartsScriptEvent.java]: missing a `$`

File: src/main/java/net/gnomeffinway/depenizen/events/PVPArenaEvents.java [hunk 2]
```diff
@@ -41,7 +43,7 @@ public void onArenaStarts(PAStartEvent event) {
         context.put("fighters", new dList(fighters));
 
         OldEventManager.doEvents(Arrays.asList
-                        ("arena starts"),
+                        ("pvparena starts"),
                 null, context);
```
Comment (mcmonkey4eva) [inline src/main/java/net/gnomeffinway/depenizen/events/PVPArenaEvents.java]: ... wot.

Comment (Fortifier42) [inline src/main/java/net/gnomeffinway/depenizen/events/PVPArenaEvents.java]: I wanted to make the pvparena event unique so it wasn't confused with other events.

Comment (mcmonkey4eva) [inline src/main/java/net/gnomeffinway/depenizen/events/PVPArenaEvents.java]: A: Fix the meta
B: Back-support the old version

**Pull request merged**